config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -81,6 +81,11 @@ presets:
     value: "false"
   - name: HOLLOW_PROXY_TEST_ARGS
     value: "--use-real-proxier=false"
+  - name: HEAPSTER_KUBELET_TEST_ARGS
+    value: "--register-with-taints=monitoring=:PreferNoSchedule"
+  - name: HEAPSTER_MACHINE_TYPE
```

Comment:
Won't it create heapster node also in the base cluster?

Reply:
This change should add an extra, tainted node to the base cluster.

Reply:
OK - that was a stupid question.